content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -834,11 +834,34 @@ commands.spawn((
 ));
 ```
 
-## Exposure Settings
+## Camera Exposure
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @superdump (Rob Swain), @JMS55, @cart</div>
 
-TODO.
+Cameras in the "real world" control how bright an image is by adjusting "exposure settings":
```

Comment:
```suggestion
In the "real world", the brightness of an image captured by a camera is determined by it's exposure: the amount of light that reaches the camera's sensor or film. This is controlled by several mechanics of the camera:
```

Reply:
I like the added clarity!

>  the amount of light that reaches the camera's sensor or film

However this actually excludes the ISO Sensitivity, which is a process that happens _as_ the light hits the sensor or film

Reply:
O, right. I forgot the classic definition of exposure excludes the ISO.  Is the edit I made better?

Reply:
Used essentially your phrasing but tweaked a couple of things.